### ISSUE TITLE
generate error

### ISSUE THREAD

**Issue by czarmax** [saleor/saleor-app-payment-stripe#214]
Hello,

I have an error when I try to run pnpm dev. During the generate phase by json-schema-compiler I have below error .

`ResolverError: Error opening file "C:\Users\PC\SALEOR\service-stripe\node_modules\.pnpm\@saleor+json-schema-compiler@0.1.2\node_modules\@saleor\.\src\schemas\definitions.json"
ENOENT: no such file or directory, open 'C:\Users\PC\SALEOR\service-stripe\node_modules\.pnpm\@saleor+json-schema-compiler@0.1.2\node_modules\@saleor\src\schemas\definitions.json'
    at ReadFileContext.callback (C:\Users\PC\SALEOR\service-stripe\node_modules\.pnpm\@saleor+json-schema-compiler@0.1.2\node_modules\@saleor\json-schema-compiler\dist\index.js:81:73346)
    at FSReqCallback.readFileAfterOpen [as oncomplete] (node:fs:306:13)
 ELIFECYCLE  Command failed with exit code 1.
 ELIFECYCLE  Command failed with exit code 1.
 ELIFECYCLE  Command failed with exit code 1.
`

Did anybody face the same ?

**Comment by czarmax** [comment]
Finally I just created docker image with alpine and it worked. Probably it's windows related.

**Comment by everestkid** [comment]
Yes I faced the same error in Windows, however it works in Linux
